apps/web/public/static/locales/en/common.json

```diff
@@ -1027,9 +1027,9 @@
   "installed_other": "{{count}} installed",
   "verify_wallet": "Verify Wallet",
   "create_events_on": "Create events on",
-  "enterprise_license": "This is an enterprise feature",
+  "enterprise_license": "This is a commercial feature",
   "enterprise_license_locally": "You can test this feature locally but not on production.",
-  "enterprise_license_sales": "To upgrade to the enterprise edition, please reach out to our sales team. If a license key is already in place, please contact support@cal.com for help.",
```

Comment:
@keithwillcode changing the English source string will trigger a new translation right?